controllers/openstackcluster_controller.go

```diff
@@ -338,10 +315,31 @@ func (r *OpenStackClusterReconciler) reconcileNetworkComponents(log logr.Logger,
 	}
 
 	if openStackCluster.Spec.ManagedAPIServerLoadBalancer {
+		err = networkingService.ReconcileControlPlaneEndpoint(openStackCluster)
```

Comment:
isn't this almost the same code as in the else just with other properties?

Reply:
I will update merging HA and non HA case.